containers/ecr-viewer/README.md

```diff
@@ -142,59 +142,30 @@ Note: The diagram omits infrastructure related to OpenTelemetry. OpenTelemetry e
 
```

Comment:
nit on the readme - when you first load the docs page, it automatically directs to the readme. Could we have a disclaimer at the top that says something like:

"If you're looking to contribute to development on the eCR Viewer, you're in the right place. If you're trying to run a deployment of the Viewer instead, please refer to our guide.md." 

Probably a good thing to have generally, since folks might come to our Github repo first regardless!